fleetdm/fleet
# Fleet UI: New manage query automations modal

## Issue
Cerra #12645 
## Description
Adds new manage automations modal

## Screenshot
<img width="633" alt="Screenshot 2023-07-13 at 11 18 14 AM" src="https://github.com/fleetdm/fleet/assets/71795832/0f05363e-e052-494c-9190-add8b2734353">

## Screenrecording
https://github.com/fleetdm/fleet/assets/71795832/3d115728-502a-474f-b375-477268320598

## PR testing instructions
1. Go to entities/queries.ts
2. Delete `sendRequest` on line 2
3. Uncomment line 9 to import mock service's `sendRequest` instead
Voila! You should see mock data in the branch's UI

# Checklist for submitter

If some of the following don't apply, delete the relevant line.


## TODO post-merge
- [ ] Added/updated tests (QA Wolf update)
- [ ] Manual QA for all new/changed functionality (Need backend ready for rebase)
- [ ] Remove mock data once the above is done
- [ ] Design decision around 10+ queries (potentially on a future iteration)
